cli/src/formatter.rs

```diff
@@ -835,6 +845,33 @@ mod tests {
         ");
     }
 
+    #[test]
+    fn test_color_formatter_ansi256() {
+        // Test the color code for each color.
+        let config = config_from_string(indoc! {"
```

Comment:
Do ANSI colors work as background colors? It'd be nice to have a test for that. If it doesn't work, perhaps we need some logic to prohibit it.

E.g. have a color like

```toml
bgpurple = { fg = 'white', bg = 'ansicolor93' }
```

Reply:
Indeed, good idea.